ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Report
+about: Create a report to help us improve.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is.-->
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+**Output**
+<!-- Copy/pasting the output is preferred, even for large error messages-->
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen.-->
+
+**Additional context**
+<!-- Add any other context about the problem here. -->
```

Comment:
I know we can add more things later, but one thing I really like from some of the other templates I've used in the past is that there's a bit here about asking for the conda environment / python version / software version.

Do you think it might be in scope to add it here? I would love to encourage folks to just give us their environment info by default (rather than having to ask for it).

Reply:
Of course I'm thinking of interchange here: https://github.com/openforcefield/openff-interchange/blob/main/.github/ISSUE_TEMPLATE/BUG_REPORT.md?plain=1#L19-L23

Reply:
yes I was also thinking about this - I'll add it now since we agree.